.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
     - env: TARGET=aarch64-linux-android RUSTFLAGS="-C target-feature=+neon"
       name: "aarch64-unknown-linux-android + NEON"
     # Linux:
-    - env: TARGET=i586-unknown-linux-gnu
+    - env: TARGET=i586-unknown-linux-gnu RUSTFLAGS="-C target-feature=+sse"
       name: "i586-unknown-linux-gnu + SSE"
```

Comment:
I think we need to add a second build bot here so that we have one with SSE and one without.